android/MobiledgeXSDKDemo/README.md

```diff
@@ -67,11 +67,11 @@ Things to try:
 - Go to settings and turn on "Show Latency Stats after session" to get a stats summary that can be copy/pasted for additional use.
 
 #### Face Detection
-The Face Detection activity provides a visual comparison of the latency offered by an Edge cloudlet vs. that of a server in the public cloud. The Edge cloudlet can be determined in a few ways, in this order of priorit: 
+The Face Detection activity provides a visual comparison of the latency offered by an Edge cloudlet vs. that of a server in the public cloud. The Edge cloudlet can be determined in a few ways, in this order of priority:
 1. The result of "Find Closest Cloudlet", if performed.
-1. Selected from within the Cloudlet Details page.
+1. Selected from within the Cloudlet Details page options menu. This also activates the "Override Edge cloudlet hostname" value in **Computer Vision Settings**.
 1. "Edge Server" entry in the **Computer Vision Settings**, if updated by the user.
-1. The default hostname: facedetection.defaultedge.mobiledgex.net.
+1. The default hostname from the provisioning data at http://opencv.facetraining.mobiledgex.net/cvprovisioning.json.
```

Comment:
Not https?

Reply:
Yeah, I need to look into getting certs for that server, and getting it set up for auto-update.